### PR TITLE
feat: integration with logdna-agent

### PR DESCRIPTION
**Pull request by imprateeksh**
### Description

logdna-agent integration has been done to validate checks in various pipelines and capture PR numbers along with repository names.

**Check the relevant boxes:**
- [ ] Bug fix (nonbreaking change that fixes an issue)
- [x] New feature (nonbreaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [x] CI related update (pipeline, etc.)

### Checklist

- [ ] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

File: module-assets/ci/run-tests.sh [hunk 1]
```diff
@@ -91,7 +97,30 @@ if [ ${IS_PR} == true ]; then
...
+      if [ -z "$MZ_LOG_DIRS" ]; then
+        export MZ_LOG_DIRS="/tmp"
+      fi
+      # This is required to send logs to logdna-agent instance
+      if [ -z "$MZ_HOST" ]; then
+        export MZ_HOST="logs.us-south.logging.cloud.ibm.com"
+      fi
+      log_location="$MZ_LOG_DIRS/test.log"
+      # Assign tags
+      export MZ_TAGS=$REPO_NAME,"PR $PR_NUM"
```
Comment (ocofaigh) [inline module-assets/ci/run-tests.sh:112]: maybe remove the space so the PR number is all one word - `"PR$PR_NUM"`

**Review by ocofaigh — CHANGES_REQUESTED** [review]
One small comment, and I think we are the good

File: module-assets/ci/run-tests.sh (hunk 1 above)
Comment (imprateeksh) [reply module-assets/ci/run-tests.sh:112]: Sure. made the change.

**Comment by imprateeksh** [comment]
> One small comment, and I think we are the good

@ocofaigh I have made the change as mentioned.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**